src/main/scala/cromwell/binding/command/Command.scala

```diff
@@ -49,7 +50,15 @@ case class Command(parts: Seq[CommandPart]) {
    * @param parameters Parameter values
    * @return String instantiation of the command
    */
-  def instantiate(parameters: Map[String, WdlValue]): String = normalize(parts.map { part => part.instantiate(parameters) }.mkString(""))
+  def instantiate(parameters: Map[String, WdlValue]): Try[String] = {
+    // http://stackoverflow.com/questions/15495678/flatten-scala-try
+    def sequence[T](xs : Seq[Try[T]]) : Try[Seq[T]] = (Try(Seq[T]()) /: xs) {
+      (a, b) => a flatMap (c => b map (d => c :+ d))
+    }
+
+    val instantiatedCommandParts = parts.map { part => part.instantiate(parameters) }
+    sequence(instantiatedCommandParts).map { _.reduceLeft(_ + _) }.map { normalize }
```

Comment:
Discussed at lunch.  :smile: 

I think I'd like to restructure this to just put a `Try` over the original (possibly exception-throwing code).  This would keep the matching logic in `CallActor` exactly the same as it is now.  Thoughts?